org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/ITable.java

```diff
@@ -975,11 +975,11 @@ default void addUITableListener(TableListener listener, Integer... eventTypes) {
   void setTableCustomizer(ITableCustomizer c);
 
   /**
-   * Container of this table, {@link org.eclipse.scout.rt.client.ui.desktop.outline.pages.IPage IPage},
+   * Container of this table, {@link IPage IPage},
    * {@link org.eclipse.scout.rt.client.ui.form.fields.tablefield.ITableField ITableField},
    * {@link org.eclipse.scout.rt.client.ui.form.fields.listbox.IListBox IListBox}, etc.
    * <p>
-   * https://bugs.eclipse.org/bugs/show_bug.cgi?id=388227
+   * <a href="https://bugs.eclipse.org/bugs/show_bug.cgi?id=388227">Bug 388227</a>
```

Comment:
Is this bug of any additional value? As far as I can see, there is not much documentation in this bug. Maybe just remove the link?